include/xleaflet/xcontrol.hpp

```diff
@@ -95,6 +98,13 @@ namespace xlf
         this->_view_module_version() = jupyter_leaflet_semver();
 
         this->options() = std::vector<std::string>();
+
+        this->options().insert(
+            this->options().end(),
+            {
+                "position"
+            }
+        );
```

Comment:
You could directly do 

```c++
this->options() = {"position"};
```

instead of 
```c++
this->options() = std::vector<std::string>();

this->options().insert(
    this->options().end(),
    {
        "position"
    }
);
```